### PR TITLE
docs(connector): adding a process overview via DBLP section

### PR DESCRIPTION
**Pull request by peiwangdb**
# Description

adding a draft section for process overview via DBLP.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Snapshots:

Include snapshots for easier review.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have already squashed the commits and make the commit message conform to the project standard.
- [ ] I have already marked the commit with "BREAKING CHANGE" or "Fixes #" if needed.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/source/user_guide/connector/DBLP.ipynb [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    "Connector is an intuitive, open-source API wrapper that speeds up development by standardizing calls to multiple APIs as a simple workflow.\n",
+    "\n",
+    "Connector provides a simple wrapper to collect structured data from different Web APIs (e.g., Twitter, Spotify), making web data collection easy and efficient, without requiring advanced programming skills.\n",
+    "\n",
+    "With Connector, you can collect data in two steps: **connect** to a website and **query** the data.\n",
+    "\n",
+    "We currently support tens of websites: https://github.com/sfu-db/DataConnectorConfigs/tree/develop/api-connectors\n",
+    "\n",
+    "You can also author your own configuration files for new websites.\n",
+    "We look forward to see your contribution to facilitate other users as well."
```
Comment (nick-zrymiak) [inline docs/source/user_guide/connector/DBLP.ipynb:25]: see -> seeing

File: docs/source/user_guide/connector/DBLP.ipynb [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    "conn = connect(\"dblp\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step3: Understand how to use the API\n",
+    "\n",
+    "info() function helps you understand what are available from the website.\n",
```
Comment (nick-zrymiak) [inline docs/source/user_guide/connector/DBLP.ipynb:79]: what are -> what is

File: docs/source/user_guide/connector/DBLP.ipynb [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    "Here, the output shows there is one table called \"publication\" is available. And to fetch the data, we have to specify the value of the \"q\" (query keyword) parameter.\n",
```
Comment (nick-zrymiak) [inline docs/source/user_guide/connector/DBLP.ipynb:80]: "publication" is available -> "publication" available

File: docs/source/user_guide/connector/DBLP.ipynb [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+   "outputs": [],
+   "source": [
+    "conn.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step4: Customize the query to collect data\n",
```
Comment (nick-zrymiak) [inline docs/source/user_guide/connector/DBLP.ipynb:97]: Step4 -> Step 4

File: docs/source/user_guide/connector/DBLP.ipynb [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+   "outputs": [],
+   "source": [
+    "await conn.query(\"publication\", q=\"CVPR 2020\", _count=2000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And you have the data ready. It is so simple."
```
Comment (nick-zrymiak) [inline docs/source/user_guide/connector/DBLP.ipynb:120]: I really like this line!!

**Review by nick-zrymiak — COMMENTED** [review]
Great work 🙌

File: docs/source/user_guide/connector/DBLP.ipynb (hunk 5 above)
Comment (peiwangdb) [reply docs/source/user_guide/connector/DBLP.ipynb:120]: :)

**Comment by peiwangdb** [comment]
Thanks for the comments @nick-zrymiak , resolved!

**Review by dovahcrow — APPROVED** [review]

**Pull request merged**